robots/fer/fer.urdf.xacro

```diff
@@ -68,4 +72,17 @@
                       arm_prefix= ""
                       connected_to= "base">
   </xacro:franka_robot>
-</robot>
+
+  <!-- ATI sensors have sampling rate as 7000 divided by an integer -->
+  <!-- Dividing by 3 is the closest to 2 times 1000 Hz which is the -->
+  <!-- control frequency of Panda -->
```

Comment:
Why do we want a frequency of 2kHz and not 1kHz ?
I though we controlled the Panda at 1kHz.

Reply:
I had a discussion with a friend of mine who was using those sensors and he suggested to me to go 2x the control frequency. We can go down to 1 kHz if you want

Reply:
Naively, I would assume that, when the frequencies do not match, what matters is the maximal delay.

Since the sensor if the fastest, the maximal delay is the period of the sensor measurement. At 7e3 / 3 ~= 2333Hz, the maximal delay would be about 429ms.

- Do we have any reason not to have the sensor give values at 7kHz (i.e. max delay of 143ms) ?

Reply:
I didn't want to go with full 7 kHz as I was worried that we might end up having throughput issues or inconsistent packet handling at the kernel level. My way of thinking is to first get the calibration setup working and then once we will want to increase precision, we might play with frequencies of measurement and maybe internal filtering. Tho I didn't pay much attention to reading the datasheet of the sensor.

Reply:
I think it is rather the opposite. We need the maximal precision for calibration. If it works, I would simply use 7kHz for calibration and something slower for other usages.
Calibration requires to perform dynamical motions with a good synchronization between the sensors. While at runtime, it is less important that we have a good synchronization, especially if we have a good calibration.